scio-core/src/main/scala/com/spotify/scio/ScioContext.scala

```diff
@@ -550,8 +611,7 @@ class ScioContext private[scio] (val options: PipelineOptions, private var artif
     options.setJobName(name)
   }
 
-  /** Close the context. No operation can be performed once the context is closed. */
-  def close(): ClosedScioContext = requireNotClosed {
+  private[scio] def prepare(): ScioContext = requireNotClosed {
```

Comment:
any reason this logic can't go in `ScioRunner` directly?

Reply:
reason is to not leak `ScioContext` internals